tests/SprykerTest/Shared/Testify/_support/Helper/ClassResolverTrait.php

```diff
@@ -44,12 +44,13 @@ protected function resolveClass(string $classNamePattern, string $moduleName): ?
     /**
      * @param string $classNamePattern
      * @param string $moduleName
+     * @param string|null $layerName
      *
      * @return string|null
      */
-    protected function resolveClassName(string $classNamePattern, string $moduleName): ?string
+    protected function resolveClassName(string $classNamePattern, string $moduleName, ?string $layerName = null): ?string
```

Comment:
`layerName` should be `application`. I guess it will be one of Glue, Zed, Yves, etc.